sympy/stats/rv.py

```diff
@@ -1639,7 +1639,7 @@ def sample(self, size=(), library='scipy', seed=None):
                 import pymc3 as pymc
 
             with pymc.Model():
-                if do_sample_pymc(self):
+                if do_sample_pymc(self) is not None:
```

Comment:
This fixes the optional dependency jobs in CI after pymc 5.21.0 was just released.